### PR TITLE
DistributionImplementation::computeQuantile fix

### LINKED ISSUE #2513: Binomial quantile computation fails on extreme example
### What happened?

The following fails on my machine:

```python
bin = ot.Binomial(1030, 0.5)
cdf = bin.computeCDF(400) # cdf = 3.8973524662436084e-13
print(bin.computeQuantile(cdf))
```
Output : [0] instead of the expeted [400]

Note that R succeeds:

```R
> qbinom(3.8973524662436084e-13, 1030, 0.5)
[1] 400
```

### How to reproduce the issue?

```shell
See above
```

### Version

master (1.23dev) (same result on pip-installed 1.22rc1)

### Operating System

Linux

### Installation media

from source

### Additional Context

See https://github.com/openturns/openturns/commit/4f3d29c83f6f37b6c6e07ddfecceac225066ffc4

### LINKED ISSUE #2548: Distribution::computeQuantile(p) can be computed for p<0 or p>1
### What happened?

Everything is in the title: I think this behavior strange. Moreover, for 1d distributions, the value differs from what `computeScalarQuantile` returns.

### How to reproduce the issue?

```shell
import openturns as ot

dist = ot.Uniform()
print(dist.computeQuantile(-1.0)[0], dist.computeScalarQuantile(-1.0)) # -1.0 3.0
print(dist.computeQuantile(2.0)[0], dist.computeScalarQuantile(2.0)) # 1.0 3.0
```

### Version

1.22

### Operating System

Linux

### Installation media

conda

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by josephmure**
- Checks for dimension 1 first.
- Extremities of the range are used only for prob (resp. 1 - prob) < cdfEpsilon_ rather than quantileEpsilon_ 
- Fixes #2513 according to the suggestion of Régis
- EDIT: also Fixes #2548 

(duplicate of #2514 which could not be reopened)

### CODE REVIEW

**Comment by josephmure** [comment]
I find the philosophy of the test `t_DistributionQuantile.py` strange: we want the quantile at 0 to be equal to the quantile at `ScalarEpsilon` and the quandtile at 1 to be equal to the quantile at `1 - ScalarEpsilon` when we are often capable of computing more accurate values.

Here is the result of the print with my branch;

```
Beta 1- [1] [1]
Burr 0+ [0] [2.22507e-308]
Burr 1- [4.5036e+15] [inf]
Dirichlet 0+ [0] [2.22507e-308]
Dirichlet 1- [1] [1]
Exponential 0+ [0] [2.22507e-308]
Exponential 1- [36.0437] [inf]
FisherSnedecor 1- [5.92577e+06] [2.09715e+06] # STRANGE: THE VALUE AT 1- SHOULD BE LOWER THAN THE VALUE AT 1
Frechet 0+ [0] [0.00141164]
Frechet 1- [4.5036e+15] [inf]
Gamma 1- [36.0437] [inf]
GeneralizedPareto 0+ [0] [2.22507e-308]
GeneralizedPareto 1- [36.0437] [inf]
Gumbel 0+ [-inf] [-6.563]
Gumbel 1- [36.0437] [inf]
Histogram 0+ [0] [2.22507e-308]
Histogram 1- [1] [1]
Laplace 1- [35.3505] [inf]
Logistic 0+ [-inf] [-708.396]
Logistic 1- [36.0437] [inf]
LogNormal 0+ [0] [5.07622e-17]
LogNormal 1- [3380.88] [inf]
LogUniform 1- [2.71828] [2.71828]
Normal 0+ [-inf] [-37.5194]
Normal 1- [8.12589] [inf]
Pareto 1- [4.5036e+15] [inf]
Rayleigh 0+ [0] [2.10954e-154]
Rayleigh 1- [8.49042] [inf]
Student 1- [170608] [inf]
Trapezoidal 1- [2] [2]
Triangular 1- [1] [1]
TruncatedNormal 1- [1] [1]
Uniform 1- [1] [1]
WeibullMax 0+ [-inf] [-708.396]
WeibullMax 1- [-2.22045e-16] [0]
WeibullMin 1- [36.0437] [inf]
Geometric 1- [52] [inf]
NegativeBinomial 1- [51] [45] # STRANGE: THE VALUE AT 1- SHOULD BE LOWER THAN THE VALUE AT 1
```

**Comment by josephmure** [comment]
I am stuck here because the remaining error is from the "Distribution.drawQuantile" docstring and only occurs on the MacOS and Windows machines, so I cannot reproduce it:

```
 File "/usr/local/lib/python3.12/site-packages/openturns/model_copula.py", line 6509, in openturns.model_copula.Distribution.drawQuantile
Failed example:
    view = View(graph)
Exception raised:
    Traceback (most recent call last):
      File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/doctest.py", line 1361, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest openturns.model_copula.Distribution.drawQuantile[8]>", line 1, in <module>
        view = View(graph)
               ^^^^^^^^^^^
      File "/usr/local/lib/python3.12/site-packages/openturns/viewer.py", line 292, in __init__
        self._ax[0].set_xlim([bb.getLowerBound()[0], bb.getUpperBound()[0]])
      File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/matplotlib/axes/_base.py", line 3664, in set_xlim
        return self.xaxis._set_lim(left, right, emit=emit, auto=auto)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/matplotlib/axis.py", line 1210, in _set_lim
        v0 = self.axes._validate_converted_limits(v0, self.convert_units)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/matplotlib/axes/_base.py", line 3585, in _validate_converted_limits
        raise ValueError("Axis limits cannot be NaN or Inf")
    ValueError: Axis limits cannot be NaN or Inf
```

**Comment by jschueller** [comment]
sometimes there are no better options than adding std::cout to debug

**Comment by josephmure** [comment]
My current hypothesis is that the default qMax is incorrectly computed on MacOS and Windows : it is supposed to be 1.0 - SpecFunc::ScalarEpsilon. If instead of 1.0 - SpecFunc::ScalarEpsilon you use 1.0 you get the same error on Linux.

**Comment by josephmure** [comment]
On my Linux machine:
```
WRN - data = class=Sample name=Unnamed implementation=class=SampleImplementation name=Unnamed size=129 dimension=2 data=[[-7.95604,8.88178e-16],[-1.93687,0.0267347],[-1.65087,0.0506437],[-1.47201,0.0731178],[-1.33912,0.0946025],[-1.23221,0.115337],[-1.14214,0.135475],[-1.0639,0.155121],[-0.994458,0.174353],[-0.931836,0.193234],[-0.874653,0.21181],[-0.821915,0.230121],[-0.77288,0.248201],[-0.726979,0.266076],[-0.683765,0.283772],[-0.642881,0.301309],[-0.604036,0.318705],[-0.566991,0.335977],[-0.531545,0.353139],[-0.49753,0.370206],[-0.464802,0.387188],[-0.433239,0.404097],[-0.402732,0.420944],[-0.373188,0.437737],[-0.344525,0.454486],[-0.316672,0.471198],[-0.289563,0.487883],[-0.263142,0.504546],[-0.237357,0.521196],[-0.212161,0.537838],[-0.187513,0.55448],[-0.163374,0.571128],[-0.13971,0.587787],[-0.11649,0.604463],[-0.0936831,0.621162],[-0.0712639,0.63789],[-0.0492072,0.654652],[-0.0274902,0.671453],[-0.0060916,0.688299],[0.0150083,0.705194],[0.0358281,0.722144],[0.0563849,0.739154],[0.0766951,0.75623],[0.0967738,0.773375],[0.116635,0.790596],[0.136294,0.807897],[0.155761,0.825284],[0.17505,0.842761],[0.194173,0.860335],[0.21314,0.87801],[0.231961,0.895792],[0.250648,0.913687],[0.26921,0.931699],[0.287656,0.949835],[0.305996,0.968101],[0.324238,0.986502],[0.342391,1.00504],[0.360463,1.02374],[0.378462,1.04258],[0.396397,1.06159],[0.414274,1.08076],[0.432102,1.10011],[0.449889,1.11965],[0.467641,1.13937],[0.485366,1.15929],[0.503071,1.17942],[0.520764,1.19976],[0.538452,1.22033],[0.556142,1.24113],[0.573841,1.26218],[0.591558,1.28347],[0.609299,1.30503],[0.627072,1.32687],[0.644884,1.34899],[0.662744,1.37142],[0.68066,1.39415],[0.698639,1.41721],[0.71669,1.44061],[0.734822,1.46437],[0.753043,1.4885],[0.771363,1.51301],[0.789791,1.53793],[0.808338,1.56328],[0.827013,1.58907],[0.845828,1.61533],[0.864793,1.64208],[0.883922,1.66935],[0.903226,1.69716],[0.922718,1.72554],[0.942414,1.75452],[0.962327,1.78413],[0.982474,1.81441],[1.00287,1.84541],[1.02354,1.87714],[1.0445,1.90968],[1.06576,1.94305],[1.08736,1.97733],[1.10932,2.01256],[1.13166,2.0488],[1.15442,2.08614],[1.17762,2.12465],[1.2013,2.16442],[1.22551,2.20553],[1.25028,2.24811],[1.27567,2.29227],[1.30172,2.33815],[1.3285,2.3859],[1.35608,2.4357],[1.38454,2.48775],[1.41397,2.54229],[1.44446,2.59958],[1.47615,2.65995],[1.50917,2.72376],[1.5437,2.79147],[1.57993,2.86362],[1.61811,2.94087],[1.65854,3.02404],[1.70159,3.11415],[1.74775,3.21255],[1.79764,3.32099],[1.85209,3.44183],[1.91228,3.5784],[1.97986,3.73558],[2.05742,3.92091],[2.14916,4.14705],[2.26283,4.43771],[2.41542,4.84614],[2.65908,5.54224],[8.12589,36.0437]]
WRN - qMin =2.22045e-16 ; qMax = 1
WRN - quantile(qMin) = class=Point name=Unnamed dimension=2 values=[-7.95604,8.88178e-16]
WRN - quantile(qMax) = class=Point name=Unnamed dimension=2 values=[8.12589,36.0437]
```

On the MacOS CI:
```
WRN - data = class=Sample name=Unnamed implementation=class=SampleImplementation name=Unnamed size=129 dimension=2 data=[[-7.95604,8.88178e-16],[-1.93687,0.0267347],[-1.65087,0.0506437],[-1.47201,0.0731178],[-1.33912,0.0946025],[-1.23221,0.115337],[-1.14214,0.135475],[-1.0639,0.155121],[-0.994458,0.174353],[-0.931836,0.193234],[-0.874653,0.21181],[-0.821915,0.230121],[-0.77288,0.248201],[-0.726979,0.266076],[-0.683765,0.283772],[-0.642881,0.301309],[-0.604036,0.318705],[-0.566991,0.335977],[-0.531545,0.353139],[-0.49753,0.370206],[-0.464802,0.387188],[-0.433239,0.404097],[-0.402732,0.420944],[-0.373188,0.437737],[-0.344525,0.454486],[-0.316672,0.471198],[-0.289563,0.487883],[-0.263142,0.504546],[-0.237357,0.521196],[-0.212161,0.537838],[-0.187513,0.55448],[-0.163374,0.571128],[-0.13971,0.587787],[-0.11649,0.604463],[-0.0936831,0.621162],[-0.0712639,0.63789],[-0.0492072,0.654652],[-0.0274902,0.671453],[-0.0060916,0.688299],[0.0150083,0.705194],[0.0358281,0.722144],[0.0563849,0.739154],[0.0766951,0.75623],[0.0967738,0.773375],[0.116635,0.790596],[0.136294,0.807897],[0.155761,0.825284],[0.17505,0.842761],[0.194173,0.860335],[0.21314,0.87801],[0.231961,0.895792],[0.250648,0.913687],[0.26921,0.931699],[0.287656,0.949835],[0.305996,0.968101],[0.324238,0.986502],[0.342391,1.00504],[0.360463,1.02374],[0.378462,1.04258],[0.396397,1.06159],[0.414274,1.08076],[0.432102,1.10011],[0.449889,1.11965],[0.467641,1.13937],[0.485366,1.15929],[0.503071,1.17942],[0.520764,1.19976],[0.538452,1.22033],[0.556142,1.24113],[0.573841,1.26218],[0.591558,1.28347],[0.609299,1.30503],[0.627072,1.32687],[0.644884,1.34899],[0.662744,1.37142],[0.68066,1.39415],[0.698639,1.41721],[0.71669,1.44061],[0.734822,1.46437],[0.753043,1.4885],[0.771363,1.51301],[0.789791,1.53793],[0.808338,1.56328],[0.827013,1.58907],[0.845828,1.61533],[0.864793,1.64208],[0.883922,1.66935],[0.903226,1.69716],[0.922718,1.72554],[0.942414,1.75452],[0.962327,1.78413],[0.982474,1.81441],[1.00287,1.84541],[1.02354,1.87714],[1.0445,1.90968],[1.06576,1.94305],[1.08736,1.97733],[1.10932,2.01256],[1.13166,2.0488],[1.15442,2.08614],[1.17762,2.12465],[1.2013,2.16442],[1.22551,2.20553],[1.25028,2.24811],[1.27567,2.29227],[1.30172,2.33815],[1.3285,2.3859],[1.35608,2.4357],[1.38454,2.48775],[1.41397,2.54229],[1.44446,2.59958],[1.47615,2.65995],[1.50917,2.72376],[1.5437,2.79147],[1.57993,2.86362],[1.61811,2.94087],[1.65854,3.02404],[1.70159,3.11415],[1.74775,3.21255],[1.79764,3.32099],[1.85209,3.44183],[1.91228,3.5784],[1.97986,3.73558],[2.05742,3.92091],[2.14916,4.14705],[2.26283,4.43771],[2.41542,4.84614],[2.65908,5.54224],[inf,inf]]
WRN - qMin =2.22045e-16 ; qMax = 1
WRN - quantile(qMin) = class=Point name=Unnamed dimension=2 values=[-7.95604,8.88178e-16]
WRN - quantile(qMax) = class=Point name=Unnamed dimension=2 values=[inf,inf]
```

In both cases, `qMax` is printed as `1` but that print seems to be inaccurate. Note that on my machine:
```python
>>> print(distribution.computeQuantile(1.0))
[inf,inf]
>>> print(distribution.computeQuantile(1.0 - ot.SpecFunc.ScalarEpsilon))
[8.12589,36.0437]
```

**Comment by josephmure** [comment]
So the latest debug yields this on MacOS:
```
WRN - In ClaytonCopula::computeQuantile: q = 1 and return value - 1 = 0
```

On LInux:
```
WRN - In ClaytonCopula::computeQuantile: q = 1 and return value - 1 = -2.22045e-16
```

Even though it reads "q = 1", the value this code is run on is `q = 1.0 - SpecFunc::ScalarEpsilon`, and the line that prompts this output is:

```c++
LOGWARN(OSS() << "In ClaytonCopula::computeQuantile: q = " << q << " and return value - 1 = " << std::exp((M_LN2 - log1p(std::pow(q, -theta_))) / theta_) - 1.0);
``` 

Apparently, the formula 
```c++
std::exp((M_LN2 - log1p(std::pow(q, -theta_))) / theta_)
```
is not as accurate on these platforms: it yields 1 on MacOS  / Windows instead of 1 - 2.22045e-16 on Linux. Do you have any idea why @jschueller @regislebrun ?

**Comment by jschueller** [comment]
no clue, maybe you could decompose the formula and see at which intermediate step it differs

**Comment by josephmure** [comment]
OK, so the issue is precisely here:

On MacOS and Windows:

```
WRN - In ClaytonCopula::computeQuantile: q - 1 = -2.22045e-16 and theta_ = 0.5
WRN - M_LN2 - log1p(std::pow(q, -theta_)) = 0
```

On Linux::
```
WRN - In ClaytonCopula::computeQuantile: q - 1 = -2.22045e-16 and theta_ = 0.5
WRN - M_LN2 - log1p(std::pow(q, -theta_)) = -1.11022e-16
```

**Comment by josephmure** [comment]
So, once again, I am stuck: how should such differences between the handling of std::pow be handled?

**Comment by mbaudin47** [comment]
> OK, so the issue is precisely at the computation of q−θ with q=1−ϵ, ϵ≈ -2.22045e-16 and θ=1/2.

If I understand correctly, the quantile formula is:

$$
F^{-1}(p) = \exp\left(\frac{\log(2) - \log\left(1 + (1 - p)^{-\theta}\right)}{\theta}\right)
$$

for any $p \in [0,1]$.

If $q = 1 - p$ is in within $10^{-8}$ of 1, then the result is $1 + \frac{q - 1}{2}$ at order 1 (because $\theta$ simplifies). This is because of the Taylor expansion of the function at point $q = 1$ (see [this link](https://www.wolframalpha.com/input?i=taylor+expansion+of+%28log%282%29+-+log%281+%2B+x%5E%28-t%29%29%29+%2F+t+at+x+%3D+1)).

The problem, however, is that the significant digits are already lost when $q$ is evaluated with the formula $q = 1 - p \approx 1$. In this case, the result is $1 - \frac{p}{2}$ at order 1. If $0 \leq p \leq 10^{-8}$, then the floating point representation of $F^{-1}(p)$ is equal to the floating point representation of $1 - \frac{p}{2}$. There is no special function involved here, so that this formula should be stable across platforms because only floating arithmetic is used.

In other words, the code at [ClaytonCopula.cxx](https://github.com/openturns/openturns/blob/30d024a438fbdd2de8eaa212a04ecf2fdf9348c4/lib/src/Uncertainty/Distribution/ClaytonCopula.cxx#L396) may be more accurate if we distinguish between the two cases: 1) if $0 \leq p \leq 1/2$, then use $p$, 2) otherwise use $q$. In both cases, an accurate formula should be used. The current implementation might be more accurate when $q \in [0,1/2]$. In other case, a formula based on $p$ should be used. If $p$ is very near zero, then a Taylor expansion will provide a stable implementation.

File: lib/src/Uncertainty/Distribution/FisherSnedecor.cxx [hunk 1]
```diff
@@ -180,10 +180,12 @@ Scalar FisherSnedecor::computeCDF(const Point & point) const
...
     const Bool tail) const
 {
-  if (getRange().getUpperBound()[0] < 0.0) return DistributionImplementation::computeScalarQuantile(prob, tail);
+  if (!((prob >= 0.0) && (prob <= 1.0)))
+    throw InvalidArgumentException(HERE) << "computeScalarQuantile expected prob to belong to [0,1], but is " << prob;
+  //if (getRange().getUpperBound()[0] < 0.0) return DistributionImplementation::computeScalarQuantile(prob, tail);
   const Scalar p = tail ? 1.0 - prob : prob;
   const Scalar q = DistFunc::qBeta(0.5 * d1_, 0.5 * d2_, p);
-  if (q >= 1.0) return getRange().getUpperBound()[0];
+  //if (q >= 1.0) return getRange().getUpperBound()[0];
```
Comment (jschueller) [inline lib/src/Uncertainty/Distribution/FisherSnedecor.cxx:188]: drop it ?

File: lib/src/Uncertainty/Distribution/Gumbel.cxx [hunk 2]
```diff
@@ -21,7 +21,7 @@
 #include <cmath>
 #include "openturns/Gumbel.hxx"
 #include "openturns/RandomGenerator.hxx"
-#include "openturns/SpecFunc.hxx"
+#include "openturns/SpecFunc.hxx"
```
Comment (jschueller) [inline lib/src/Uncertainty/Distribution/Gumbel.cxx:24]: whitespace

File: lib/test/t_FisherSnedecor_std.cxx [hunk 3]
```diff
@@ -121,13 +121,14 @@ int main(int, char *[])
     CDFgrFD[1] = (FisherSnedecor(distribution.getD1(), distribution.getD2() + eps).computeCDF(point) -
                   FisherSnedecor(distribution.getD1(), distribution.getD2() - eps).computeCDF(point)) / (2.0 * eps);
     fullprint << "cdf gradient (FD)=" << CDFgrFD << std::endl;
-    Point quantile = distribution.computeQuantile( 0.95 );
+    Point quantile = distribution.  computeQuantile( 0.95 );
```
Comment (jschueller) [inline lib/test/t_FisherSnedecor_std.cxx:124]: whitespace

File: python/test/t_InverseWishart_std.expout [hunk 4]
```diff
@@ -27,7 +27,7 @@ cdf(quantile)=0.950000
 InverseSurvival= class=Point name=Unnamed dimension=1 values=[0.0903302]
 Survival(inverseSurvival)=0.950000
 entropy=-0.369531
-Minimum volume interval= [0.0458714, 0.877774]
+Minimum volume interval= [0, 0.872999]
```
[the remark this answers is not in the document]
Comment (jschueller) [reply python/test/t_InverseWishart_std.expout:30]: maybe the levelsetmesher now avoids some region that was previously allowed with clipping of the probability

File: lib/src/Uncertainty/Distribution/FisherSnedecor.cxx (hunk 1 above)
Comment (josephmure) [reply lib/src/Uncertainty/Distribution/FisherSnedecor.cxx:188]: OK

File: lib/src/Uncertainty/Distribution/Gumbel.cxx (hunk 2 above)
Comment (josephmure) [reply lib/src/Uncertainty/Distribution/Gumbel.cxx:24]: OK

File: lib/test/t_FisherSnedecor_std.cxx (hunk 3 above)
Comment (josephmure) [reply lib/test/t_FisherSnedecor_std.cxx:124]: OK

File: python/test/t_InverseWishart_std.expout (hunk 4 above)
[the remark this answers is not in the document]
Comment (josephmure) [reply python/test/t_InverseWishart_std.expout:30]: I don't understand, what code did you run?

[the remark this answers is not in the document]
Comment (jschueller) [reply python/test/t_InverseWishart_std.expout:30]: I ran distribution.computeProbability on the old & new intervals from the diff

[the remark this answers is not in the document]
Comment (josephmure) [reply python/test/t_InverseWishart_std.expout:30]: I reran this and I get a correct value:
```python
interval, threshold = distribution.computeMinimumVolumeIntervalWithMarginalProbability(
    0.95
)
print("Minimum volume interval=", interval)
print("probability = ", distribution.computeProbability(interval))
```

Output:
```
Minimum volume interval= [0, 0.872999]
probability =  0.9500000000000002
```

[the remark this answers is not in the document]
Comment (jschueller) [reply python/test/t_InverseWishart_std.expout:30]: yes, sorry looks I mistyped the digits

[the remark this answers is not in the document]
Comment (jschueller) [reply python/test/t_InverseWishart_std.expout:30]: all good then

**Review by jschueller — APPROVED** [review]

File: lib/src/Uncertainty/Distribution/ClaytonCopula.cxx [hunk 5]
```diff
@@ -385,14 +385,19 @@ Point ClaytonCopula::computeQuantile(const Scalar prob,
...
+  if (prob == 0.0) return tail ? getRange().getUpperBound() : getRange().getLowerBound();
+  if (prob == 1.0) return tail ? getRange().getLowerBound() : getRange().getUpperBound();
   // Independent case
-  if (theta_ == 0.0) return Point(2, std::sqrt(q));
+  if (theta_ == 0.0) return Point(2, std::sqrt(tail ? 1.0 - prob : prob));
   // General case
-  return Point(2, std::exp((M_LN2 - log1p(std::pow(q, -theta_))) / theta_));
+  if (tail)
+  {
+    if (prob < std::sqrt(SpecFunc::ScalarEpsilon)) return Point(2, 1.0 - prob / 2.0);
```
Comment (mbaudin47) [inline lib/src/Uncertainty/Distribution/ClaytonCopula.cxx:396]: The code seems structurally correct from my point of view. But it introduces 4 different ways to return a value. Hence, this particular commit should introduce 4 different unit tests which individually check that each branch produce the correct value. Otherwise, a new bug might be introduced by this PR without being noticed by the current machinery. The initial problem was a slight numerical difference on different machines, which is much less a significant problem in general.

**Review by mbaudin47 — CHANGES_REQUESTED** [review]

**Review by regislebrun — APPROVED** [review]

File: lib/src/Uncertainty/Distribution/ClaytonCopula.cxx (hunk 5 above)
Comment (josephmure) [reply lib/src/Uncertainty/Distribution/ClaytonCopula.cxx:396]: I agree. Done.

**Comment by jschueller** [comment]
I guess it settles #2548

**Comment by josephmure** [comment]
> I guess it settles #2548

Indeed, I have updated the initial post to reflect this.

**Comment by jschueller** [comment]
do you know if @mbaudin47 is done reviewing ?

**Review by sofianehaddad — APPROVED** [review]
